Your pull request does not meet our code style rules. Pylint summary:
```
************* Module conf.py
doc/sphinx/conf.py.in:18:0: W0611: Unused import os (unused-import)
************* Module espressomd.lbboundaries
src/python/espressomd/lbboundaries.py:21:0: W0611: Unused import math (unused-import)
```

You can generate these warnings with `maintainer/CI/fix_style.sh`. This is the same command that I have executed to generate the log above.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -35,3 +35,12 @@ repos:
         always_run: false
         exclude: '.*\.(sh|py|sh\.in|cmakein)|.git'
         types: [file, executable]
+
+    -   id: pylint
+        name: pylint
+        entry: sh maintainer/lint/pylint.sh
+        language: system
+        always_run: false
+        files: '.*\.py'
+        args: ["--score=no", "--reports=no", "--output-format=text"]
+        log_file: pylint.log

--- a/doc/sphinx/conf.py.in
+++ b/doc/sphinx/conf.py.in
@@ -15,7 +15,6 @@
 # serve to show the default.
 
 import sys
-import os
 import sphinx
 
 # If extensions (or modules to document with autodoc) are in another directory,

--- a/maintainer/CI/fix_style.sh
+++ b/maintainer/CI/fix_style.sh
@@ -28,37 +28,28 @@ fi
 maintainer/lint/pre_commit.sh run --all-files
 pre_commit_return_code="${?}"
 
-if [ "${CI}" != "" ]; then
-    git --no-pager diff > style.patch
-    maintainer/gh_post_style_patch.py || exit 1
-fi
 git diff-index --quiet HEAD --
-if [ "${?}" = 1 ]; then
-    if [ "${CI}" != "" ]; then
-        echo "Failed style check. Download ${CI_JOB_URL}/artifacts/raw/style.patch to see which changes are necessary." >&2
-    else
-        echo "Failed style check" >&2
+git_diff_return_code="${?}"
+if [ "${pre_commit_return_code}" = 1 ]; then
+    if [ "${git_diff_return_code}" = 1 ]; then
+        if [ "${CI}" != "" ]; then
+            git --no-pager diff > style.patch
+            maintainer/gh_post_style_patch.py || exit 1
+            echo "Failed style check. Download ${CI_JOB_URL}/artifacts/raw/style.patch to see which changes are necessary." >&2
+        else:
+            echo "Failed style check." >&2
+        fi
     fi
-    exit 1
-else
-    if [ "${pre_commit_return_code}" = 0 ]; then
-        echo "Passed style check"
-    else
-        echo "Failed style check" >&2
+    if [ -f "pylint.log" ]; then
+        if [ "${CI}" != "" ]; then
+            errors=$(grep -Pc '^[a-z]+/.+?.py:[0-9]+:[0-9]+: [CRWEF][0-9]+:' pylint.log)
+            maintainer/gh_post_pylint.py "${errors}" pylint.log || exit 1
+            echo "Failed pylint check: ${errors} errors" >&2
+        else
+            echo "Failed pylint check." >&2
+        fi
     fi
 fi
 
-maintainer/lint/pylint.sh --score=no --reports=no --output-format=text src doc maintainer testsuite samples | tee pylint.log
-errors=$(grep -Pc '^[a-z]+/.+?.py:[0-9]+:[0-9]+: [CRWEF][0-9]+:' pylint.log)
 
-if [ "${CI}" != "" ]; then
-    maintainer/gh_post_pylint.py "${errors}" pylint.log || exit 1
-fi
-if [ "${errors}" != 0 ]; then
-  echo "Failed pylint check: ${errors} errors" >&2
-  exit 1
-else
-  echo "Passed pylint check"
-fi
-
-exit 0
+exit ${pre_commit_return_code}

